core/src/main/java/org/apache/accumulo/core/client/summary/SummarizerConfiguration.java

```diff
@@ -271,7 +271,7 @@ public Builder addOptions(String... keyValuePairs) {
      * @see SummarizerConfiguration#getOptions()
      */
     public Builder addOptions(Map<String,String> options) {
-      options.entrySet().forEach(e -> addOption(e.getKey(), e.getValue()));
+      options.forEach((key, value) -> addOption(key, value));
```

Comment:
You can even go a step further with this one:
```suggestion
      options.forEach(this::addOption);
```

Reply:
done